sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_1.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_1.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_4.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_4.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_6.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_6.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_7.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_7.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_9.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_9.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_10.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_10.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_13.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
sed: can't read /home/tomasc/genome_analysis_2021/analysis/prokka/gff/binning_13.gff: No such file or directory
[E::hts_open_format] Failed to open file "/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam" : No such file or directory
  [Errno 2] could not open alignment file `/home/tomasc/genome_analysis_2021/analysis/mapping/bin_.bam`: No such file or directory
  [Exception type: FileNotFoundError, raised in libcalignmentfile.pyx:941]
